wagtail/users/templates/wagtailusers/users/create.html

```diff
@@ -40,7 +40,7 @@
                 <ul class="fields">
                     {% include "wagtailadmin/shared/field_as_li.html" with field=form.is_superuser %}
                     {% include "wagtailadmin/shared/field_as_li.html" with field=form.groups %}
-                    <li><input type="submit" value='{% trans "Add user" %}' class="button" /></li>
+                    <li><input type="submit" value='{% trans "Add user" as add_user_button_label %}{{ add_user_button_label|force_escape }}' class="button" /></li>
```

Comment:
Hi @benoitvogel, thank you for taking the time to contribute to Wagtail. 

Instead of escaping the translation value it would be more convenient to reconsider the input submit to be a button, for eg: `<button class="button">{% trans 'Add user' %}</button>`. This way we wouldn't need to escape the value and just render it as text within the element. By default the button will have the submit type.